api/MediaCapabilities.json

```diff
@@ -0,0 +1,207 @@
+{
+  "api": {
+    "MediaCapabilities": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaCapabilities",
+        "description": "MediaCapabilities Interface",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "66"
+          },
+          "chrome_android": {
+            "version_added": "66"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "63"
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": "55"
+          },
+          "opera_android": {
+            "version_added": "55"
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
+          "deprecated": false
+        }
+      },
+      "encodingInfo": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaCapabilities/encodingInfo",
+          "description": "mediaCapabilities.encodingInfo() Method",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "67",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "chrome://flags/#enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
```

Comment:
This information applies to both `webview_android` and 'chrome_android`.

For what it's worth, I don't prioritize documenting flags because implementation and usage is an inconsistent mess. (I'm hoping to improve that in the future.)